.gitignore

```diff
@@ -103,3 +103,6 @@ ENV/
 .idea/
 crawl-data/
 output/
+
+# vscode
```

Comment:
The comment isn't really helpful? Should be better (or similar)
```
# Visual Studio Code configuration folder
```

+1 otherwise

Reply:
Seems reasonable. I guess `vscode` is only meaningful if that's the editor you use all the time. Good call :-)